packages/docusaurus-theme-classic/src/theme/MDXComponents/Ul/index.tsx

```diff
@@ -11,7 +11,11 @@ import type {Props} from '@theme/MDXComponents/Ul';
 
 import styles from './styles.module.css';
 
-function transformUlClassName(className?: string): string {
+function transformUlClassName(className?: string): string | undefined {
+  // Fix https://github.com/facebook/docusaurus/issues/9098
+  if (typeof className === 'undefined') {
```

Comment:
using typeof because if props.className is '' we probably still want to have `class=''` in the final html, although it's a weird edge-case